package.json

```diff
@@ -23,20 +23,19 @@
     "url": "https://github.com/scottcorgan/tap-spec/issues"
   },
   "dependencies": {
-    "chalk": "^1.0.0",
+    "chalk": "^2.4.1",
     "duplexer": "^0.1.1",
-    "figures": "^1.4.0",
-    "lodash": "^3.6.0",
```

Comment:
Wouldn't this PR cleaner if just "lodash" was updated?

Reply:
I do think it would be easier to just update lodash. The, we can do a major release with lodash removed. There's not _too_ much gain to removing, but I am open to removing it.

Reply:
I can understand the argument that it would be 'cleaner' to just remove lodash in this PR. However, the context (for me, at least) is that lodash is causing an npm audit warning. Part of that push is to ensure you're using up-to-date modules. I went through and reviewed the changes in these updated modules, and nothing should break functionality that's being used in tap-spec.

Reply:
Is this a blocker? Although I genuinely feel that updating to the latest version of the dependencies is in the spirit of this change, if that's the blocking issue with this PR, I will back it out to get this merged. Please let me know.